audformat/core/table.py

```diff
@@ -61,6 +61,48 @@ def __init__(
         self._db = None
         self._id = None
 
+    def __add__(self, other: typing.Self) -> typing.Self:
+        r"""Create new table by combining two tables.
+
+        The new table contains index and columns of both tables.
+        Missing values will be set to ``NaN``.
+
+        If table is conform to
+        :ref:`table specifications <data-tables:Tables>`
+        and at least one table is segmented,
+        the output has a segmented index.
+
+        Columns with the same identifier are combined to a single column.
+        This requires that:
+
+        1. both columns have the same dtype
+        2. in places where the indices overlap the values of both columns
+           match or one column contains ``NaN``
+
+        Media and split information,
+        as well as,
+        references to schemes and raters are discarded.
+        If you intend to keep them,
+        use ``update()``.
+
+        Args:
+            other: the other table
+
+        Raises:
+            ValueError: if columns with the same name have different dtypes
+            ValueError: if values in the same position do not match
```

Comment:
```suggestion
            ValueError: if values in the same position do not match
            ValueError: if level and dtypes of indices do not match
```

Reply:
Yes, I just realized that we will no longer be able to combine all `MiscTable` as we can with `Table`. So I put WIP and will add some tests.